jerry-core/ecma/operations/ecma-bigint.c

```diff
@@ -147,7 +149,11 @@ ecma_bigint_parse_string (const lit_utf8_byte_t *string_p, /**< string represena
         digit = (ecma_bigint_digit_t) (character - (LIT_CHAR_LOWERCASE_A - 10));
       }
     }
-
+    if (ecma_string_trim_front (string_p, string_end_p) == string_end_p)
```

Comment:
this needs to be `else if` after `if (character >= LIT_CHAR_LOWERCASE_A && character <= LIT_CHAR_LOWERCASE_F)`, otherwise we check spaces all the time, which is a considerable perf loss.